typescript/forms.d.ts

```diff
@@ -333,6 +333,7 @@ export type NumberElement = {
   minNumber?: number
   maxNumber?: number
   isSlider: boolean
+  isCurrency?: boolean
```

Comment:
```suggestion
  displayAsCurrency?: boolean
```

This is the name of the property on the calculation element. Not sure if it makes as much sense but throwing it in the ring anyway. Happy to merge if you prefer `isCurrency`.

Reply:
So my initial thought here was that it would be a competing property with `isSlider` so I wanted to name it consistently.  However, on reflection and confirmation with Blake the two can live together happily.  So I am ok with using `displayAsCurrency`.  My only lame objection is that I've written a heap (a few lines) of code expecting `isCurrency`.  I will go with `displayAsCurrency`.

Reply:
Can we also add some typedocs, I know none of the other properties have them but got start somewhere.